vulnerabilities/helpers.py

```diff
@@ -164,3 +165,34 @@ def __lt__(self, other):
         )
 
     return affected_package_with_patched_package_objects
+
+
+def split_markdown_front_matter(lines: str) -> Tuple[str, str]:
+    r"""
+    Split text into markdown front matter and the markdown body
+    Return ("", text) for text with non existing front matter
+
+    >>> text='''---
+    ... title: DUMMY-SECURITY-2019-001
+    ... description: Incorrect access control.
+    ... cves: [CVE-2042-1337]
+    ... ---
+    ... # Markdown starts here
+    ... '''
+    >>> split_markdown_front_matter(text)
+    ('title: DUMMY-SECURITY-2019-001\ndescription: Incorrect access control.\ncves: [CVE-2042-1337]', '# Markdown starts here\n')
+    """
+    fmlines = []
+    mdlines = []
+    splitter = mdlines
+
+    lines = lines.replace("\r\n", "\n")
+    for index, line in enumerate(lines.split("\n")):
+        if index == 0 and line.strip().startswith("---"):
+            splitter = fmlines
+        elif line.strip().startswith("---"):
```

Comment:
You should test equality, not start.

Reply:
I on some implementations it says the separator could be 3 dashes or more. For now, let's settle with 3. We'll look at it if/when some anomaly arises.